packages/lib/gpcircuits/circuits/proto-pod-gpc.circom

```diff
@@ -28,6 +30,15 @@ template ProtoPODGPC (
     // bitfields).
     MAX_ENTRIES,
 
+    // Indicates the maximum number of list entries for the list membership check.
+    MAX_LIST_ENTRIES,
+
+    // Indicates the maximum number of tuples to generate.
+    MAX_TUPLES,
+
+    // Indicates the arity of the tuples.
+    ARITY,
```

Comment:
TUPLE_ARITY?
Actually I'd suggest maybe using TUPLE_SIZE.  As a non-mathematician remind myself what "arity" meant.  It's correct, just not a commonly-used term in software, in my experience.